docs/querying/api.md

```diff
@@ -1031,6 +1031,36 @@ $ curl http://localhost:9090/api/v1/status/tsdb
 
 *New in v2.15*
 
+### WAL Replay Stats
+
+The following endpoint returns information about the WAL replay:
+
+```
+GET /api/v1/status/walreplay
+```
+
+**first**: The first segment needed to be replayed.
+**last**: The last segment needed to be replayed.
+**read**: The latest segment replayed.
+
+```json
+$ curl http://localhost:9090/api/v1/status/walreplay
+{
+  "status": "success",
+  "data": {
+    "first": 1076,
```

Comment:
I like it! Should that be an addition, or completely replace the `first`, `read`, and last fields? I intentionally wrote this endpoint to expose more information than just the starting page needs in case anyone else would like to access the data, but I could support either direction.

Reply:
Replace IMO (: